src/VisualStudio/CSharp/Impl/CSharpVSResources.resx

```diff
@@ -640,4 +640,7 @@
   <data name="Automatically_complete_statement_on_semicolon" xml:space="preserve">
     <value>Automatically complete statement on semicolon</value>
   </data>
+  <data name="Prefer_null_check_over_type_check" xml:space="preserve">
+    <value>Prefer null check over type check</value>
```

Comment:
this is inconsistent `'null'` should be in quotes.  Also... this feels like a duplicated string.  Can we unify?

Reply:
I think the duplication is tracked by https://github.com/dotnet/roslyn/issues/52343. This is not for this analyzer only. :(